doc/sphinx-guides/source/developers/containers.rst

```diff
@@ -1,6 +1,4 @@
-=================================
 Docker, Kubernetes, and OpenShift
-=================================
```

Comment:
We use the `===` stuff to create headers. Why are you removing them?

Reply:
I mistook it for a merge conflict syntax. thank you for catching that